web/modules/custom/farm_page/src/Services/FarmPageService.php

```diff
@@ -29,6 +30,27 @@ public function getStoreIdByUserId($uid) {
     return $store_id;
   }
 
+  /**
+   * Get Taxonomy term of an Object from taxonomy_id .
```

Comment:
Common doesn't make sense.

Reply:
fixed to "Get Taxonomy term by taxonomy_id"

Reply:
Change it to: Get term by tid.

Reply:
Changed